develop/smart-contracts/wasm-ink.md

```diff
@@ -118,17 +117,15 @@ There are two types of messages:
 - **Immutable messages (`&self`)** - these messages can only read the contract's state and cannot modify it
 - **Mutable messages (`&mut self`)** - these messages can read and modify the contract's state
 
-!!!note
-    `&self` is a reference to the contract's storage.
+In the example above, `&self` is a reference to the contract's storage.
 
 Example:
 
 ```rust
 --8<-- 'code/develop/smart-contracts/wasm-ink/messages.rs'
 ```
 
-!!!note
-    In the example above, `my_getter` is an immutable message that reads state, while `my_setter` is a mutable message that updates state.
+In the example above, `my_getter` is an immutable message that reads state, while `my_setter` is a mutable message that updates state.
```

Comment:
Same here

Reply:
If knowing this is essential to understanding the code snippet, then it should be added to the code as a comment